src/features/PassSurvey/ui/EntityTimer.tsx

```diff
@@ -67,6 +67,34 @@ export const EntityTimer = ({ entityTimerSettings }: Props) => {
     return `${formatTimerTime(timeToLeft)} remaining`;
   };
 
+  const checkLessThan10Mins = (): boolean => {
+    if (!groupStartAt) {
+      return false;
+    }
+
+    const timeToLeft = getTimeToLeft(new Date(groupStartAt));
+
+    if (!timeToLeft) {
+      return false;
+    }
+
+    const timeLeftInMs = getMsFromHours(timeToLeft.hours) + getMsFromMinutes(timeToLeft.minutes);
+
+    const SEC = 1000;
+
+    const MIN = SEC * 60;
```

Comment:
It would be nice to have SECOND and MINUTE constants as global constants shared across the project

Reply:
Done